var/spack/repos/builtin/packages/openfoam-org/package.py

```diff
@@ -134,13 +135,14 @@ class OpenfoamOrg(Package):
     #
 
     def setup_environment(self, spack_env, run_env):
+        # TODO: patch etc/bashrc so we can create a setup as per openfoam-com
```

Comment:
Is this comment required now that https://github.com/spack/spack/pull/6686 has been merged?

Reply:
Yes, it still doesn't seem to work. One guess is that there is somewhere in the whole etc/bashrc chain (from the openfoam-org upstream content) that is returning a non-zero exit code (maybe an `unset` instead of `unset -f` somewhere).

The comment is a good reminder that cut-and-paste may not really work, or at least that it needs more attention than simply copying across from the openfoam-com package.

Reply:
... i guess the comment is bit cryptic though (but at least it's being read!)

Reply:
IMO it should be updated to reflect that the current patch on bashrc is insufficient (otherwise someone might think this comment is stale since a patch exists now for it).

Reply:
Fair enough. I'll make the changes. Will you merge?